tests/e2e/tracer/conftest.py

```diff
@@ -0,0 +1,21 @@
+import pytest
+
+from tests.e2e.tracer.infrastructure import TracerStack
+from tests.e2e.utils.infrastructure import deploy_once
+
+
+@pytest.fixture(autouse=True, scope="module")
+def infrastructure(request: pytest.FixtureRequest, tmp_path_factory: pytest.TempPathFactory, worker_id: str):
+    """Setup and teardown logic for E2E test infrastructure
+
+    Parameters
```

Comment:
Should we update parameters docstring to reflect input?

Reply:
YES! done now.